### ISSUE TITLE
bug when trying to get the schedule for rælingen, norway (min_renovasjon)

### ISSUE THREAD

**Issue by randriksen** [mampfes/hacs_waste_collection_schedule#364]
I'm getting a weird bug when trying to get the schedule for Rælingen (Norway) through min_renovasjon

the error in HA is 
```
Logger: waste_collection_schedule.scraper
Source: custom_components/waste_collection_schedule/waste_collection_schedule/scraper.py:143
Integration: waste_collection_schedule ([documentation](https://github.com/mampfes/hacs_waste_collection_schedule#readme))
First occurred: 11:20:25 (1 occurrences)
Last logged: 11:20:25

fetch failed for source Min Renovasjon: Traceback (most recent call last): File "/config/custom_components/waste_collection_schedule/waste_collection_schedule/scraper.py", line 141, in fetch entries = self._source.fetch() File "/config/custom_components/waste_collection_schedule/waste_collection_schedule/source/minrenovasjon_no.py", line 68, in fetch icon = self._icon_map[re.sub(r"^.*?/(\w+)\.\w{3,4}$", "\\1", f['Ikon'])] KeyError: 'matrestavfallplast'
```

this can be tested with:

```
waste_collection_schedule:
  sources:
    - name: minrenovasjon_no
      args:
        street_name: "Smestadbråtan"
        house_number: "1"
        street_code: "13766"
        county_id: "3027"
```

i'm thinking that there's a problem with the fractions, this one is not an alternative: 'matrestavfallplast'

**Comment by randriksen** [comment]
found the solution, made a pull request

**Comment by mampfes** [comment]
fixed with #365
